README.md

```diff
@@ -684,50 +684,56 @@ In case you want to use a different spinner type you can modify
 `rustic-display-spinner`.([Available spinner
 types](https://github.com/Malabarba/spinner.el/blob/master/spinner.el#L104)).
 
-## inline-documentation
+## rust docs in org-mode
 
-It is possible to read rust documentation inside
-Emacs! This currently requires LSP-mode and cargo.  ![Rustic-doc
+It is possible to read rust documentation inside Emacs! This currently
+requires LSP-mode and cargo.  ![Rustic-doc
 example](img/rustic-doc.png)
 
-### Prequisites
-
-* Install Pandoc, preferably at least version 2.11, as it will give somewhat nicer generated documentation: https://pandoc.org/installing.html
-* Install cargo
-  https://doc.rust-lang.org/cargo/getting-started/installation.html
-* Install helm-ag https://github.com/emacsorphanage/helm-ag (Optional,
-  but highly recommended)
-* If you do not have them, you will be prompted to install `fd-find`,
-  `ripgrep` and `cargo-makedocs` when you run `rustic-doc-setup`.
-    * `ripgrep` is optional but highly recommended.
-    * If helm-ag and ripgrep is installed, those will be used by
-      default.
-    * If only ripgrep is installed, it will be used with the emacs
-      `grep` command.
-    * If neither is installed, the emacs `grep` command will use
-      `grep`, like in the good old days.
-    * You can change this by providing your own search function by
-      changing `rustic-doc-search-function`.
+### Prerequisites
+
+Required:
+
+- [pandoc](https://pandoc.org/installing.html) (preferably at least version 2.11, as it will give somewhat nicer generated documentation)
+- [cargo](https://doc.rust-lang.org/cargo/getting-started/installation.html)
+- cargo-makedocs
```

Comment:
Could add a link to https://github.com/Bunogi/cargo-makedocs and say "installed automatically if not found when running `rustic-doc-setup` to this and fd-find.

Reply:
Done.